web/app/js/components/MetricsTable.jsx

```diff
@@ -59,6 +59,15 @@ const columnDefinitions = (resource, namespaces, onFilterClick, showNamespaceCol
     }
   ];
 
+  let meshedColumn = {
+    title: formatTitle("M", "Meshed"),
```

Comment:
I think I'd sliiightly prefer "Mesh" as the abbreviated column title instead of "M"... the other abbreviations admittedly are terse, but Meshed is definitely not a standard enough term that M would be very clear

Reply:
That makes sense. What do you think about both columns being "Mesh" or "Meshed"? I am thinking maybe having the full title may be descriptive regardless of how the column is displayed?